examples/freely_decaying_mediterraneum.jl

```diff
@@ -0,0 +1,119 @@
+using GLMakie
+using Oceananigans
+using Oceananigans: architecture
```

Comment:
Might be good to figure out how to get rid of this so users don' thave to worry

Reply:
agreed